libmscore/style.cpp

```diff
@@ -949,7 +949,7 @@ static const StyleType styleTypes[] {
       { Sid::glissandoFontFace,             "glissandoFontFace",            "FreeSerif" },
       { Sid::glissandoFontSize,             "glissandoFontSize",            QVariant(8.0) },
       { Sid::glissandoFontSpatiumDependent, "glissandoFontSpatiumDependent", true },
-      { Sid::glissandoFontStyle,            "glissandoFontStyle",           int(FontStyle::Italic) },
+      { Sid::glissandoFontStyle,            "glissandoFontStyle",           int(FontStyle::Bold) },
```

Comment:
Not sure sabout this change though, I see no reason to change the default from italic to bold here?

Reply:
Oh! Default should be italic! Right! Why didn't I realize that?
SO I'VE BEEN LOOKING AT BOLD GLISSANDO TEXTS FOR MONTHS IN MUSESCORE AND HAVEN'T NOTICED SOMETHING WRONG ABOUT THAT??!!

Reply:
I've told you about a million times not to exagerate

Reply:
Really? I'm afraid this is the first time I hear you say it. Sorry! XD
Anyway I was just exclaiming and joking. I won't do that anymore!

Reply:
I was just kidding. And contradicting myself too ;-)

Reply:
Well...